titan-options.php

```diff
@@ -240,6 +240,80 @@ function regala_create_options() {
         'type' => 'upload',
     ) );
     
+    /*
+    *   Paging Navigation
+    */
+    $navigation = $titan->createThemeCustomizerSection( array(
+        'name' => __( 'Paging Navigation', 'regala' ),
+        'panel' => __( 'Theme Options & Colors', 'regala' ),
+    ) );
+    
+    $navigation->createOption( array(
+        'name' => __( 'Background Color Navigation', 'regala' ),
+        'id' => 'navigation_bg_color',
+        'type' => 'color',
+        'default' => '#222',
+        'css' => '.paging-navigation { background: value }',
+    ) );
+    
+    $navigation->createOption( array(
+        'name' => __( 'Font Navigation', 'regala' ),
+        'id' => 'navigation_fonts',
+        'type' => 'font',
+    	   'default' => array(
+	       'color' => '#999',
+	       'line-height' => '1.6em',
+	       'font-size' => '16px',
+	       'font-family' => 'Playfair Display',
+           'font-weight' => '900',
+           'font-style' => 'italic',
+	   ),
+	   'desc' => __( 'Select a Style', 'regala' ),
+	   'css' => 'ul.page-numbers a.page-numbers, ul.page-numbers span{ value }',
+    ) );
+    
+    /*
+    *   Color option for footer widgets
+    */
+    
+    $footer = $titan->createThemeCustomizerSection( array(
+        'name' => __( 'Footer Widgets Area', 'regala' ),
+        'panel' => __( 'Theme Options & Colors', 'regala' ),
+    ) );
+    
+    $footer->createOption( array(
+        'name' => __( 'Background Color', 'regala' ),
+        'id' => 'footer_bg_color',
+        'type' => 'color',
+        'default' => '#FFFFFF',
+        'css' => '.site-footer { background: value }',
+    ) );
+    
+    $footer->createOption( array(
+        'name' => __( 'Text Color', 'regala' ),
+        'id' => 'footer_text_color',
+        'type' => 'color',
+        'default' => '#999',
+        'css' => '.footer-widgets { color: value }',
+    ) );
+    
+    $footer->createOption( array(
+        'name' => __( 'Text Link Color', 'regala' ),
+        'id' => 'footer_link_color',
+        'type' => 'color',
+        'default' => '#DADFE1',
+        'css' => 'a, a:link, a:visited { color: value }',
```

Comment:
Change this, the selector here will modify **all** the link styles instead of just the links inside the widget area